libraries/AP_AHRS/AP_AHRS_View.h

```diff
@@ -190,4 +200,23 @@ class AP_AHRS_View
     } trig;
 
     float y_angle;
+
```

Comment:
I'd like to see the variables and logic of unspin be broken out into a separate class which is only instantiated if needed. We don't want to pay for this on all vehicles.